src/exclusion/common.hpp

```diff
@@ -35,7 +35,8 @@ struct object_set {
 
 struct rule_policy {
     filter_mode mode{filter_mode::none};
-    std::unordered_set<const ddwaf_object *> objects{};
+    std::string_view action;
+    std::unordered_set<const ddwaf_object *> objects;
```

Comment:
could have a more descriptive name

Reply:
I'll rename it to `action_override` as it's called elsewhere.